src/nvim/spell.c

```diff
@@ -4292,23 +4289,21 @@ static void suggest_trie_walk(suginfo_T *su, langp_T *lp, char_u *fword, bool so
                     && utf_iscomposing(utf_ptr2char(fword
                                                     + sp->ts_fcharstart))) {
                   sp->ts_score -= SCORE_SUBST - SCORE_SUBCOMP;
-                } else if (!soundfold
-                           && slang->sl_has_map
+                } else if (!soundfold && slang->sl_has_map
                            && similar_chars(slang,
-                                            mb_ptr2char(tword
-                                                        + sp->ts_twordlen
-                                                        - sp->ts_tcharlen),
-                                            mb_ptr2char(fword
-                                                        + sp->ts_fcharstart))) {
+                                            utf_ptr2char(tword + sp->ts_twordlen
+                                                         - sp->ts_tcharlen),
+                                            utf_ptr2char(fword +
```

Comment:
The `+` should be on the next line if this has to be split.

Reply:
Done in latest commit. Note that the ) { were move to a new line to stay within line length limits. Hope they were aligned correctly.